framework_tvb/tvb/core/neotraits/forms.py

```diff
@@ -747,7 +747,7 @@ def validate(self):
         return self.form.validate()
 
     def __str__(self):
-        return jinja_env.get_template(self.template).render(form=self.form)
+        return jinja_env.get_template(self.template).render(adapter_form=self.form)
```

Comment:
This is better hidden, but somehow the same backwards dependency that we tried to get rid in your previous PR. I will mark with a TODO for later